index.js

```diff
@@ -158,3 +160,83 @@ bot.onText(/\/meetups/, async msg => {
   );
   bot.sendMessage(msg.chat.id, out);
 });
+
+//Feature to save links and articles
+mongoose.connect("mongodb://localhost:27017/saved");
+const savedSchema = new Schema({
```

Comment:
Please use a better naming declaration for the Schema. It helps in understanding its purpose.